{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/admin/auth/user/change_list.html

```diff
@@ -1 +1,6 @@
 {% extends "admin/change_list.html" %}
+
+{# TODO Discuss this change in the CMS instead of overriding it in a template. #}
```

Comment:
Does this template do anything at all?

Reply:
it's overwriting the CMS template this is:
```{% extends "admin/change_list.html" %}


{% block object-tools-items %}
    <a class="btn" href="{% url 'admin:auth_user_invite' %}">Invite user</a>
    {{block.super}}
{% endblock %}
```

Reply:
Delete it from both CMS and PT. If we want to add back user inviting we can bring it back at a project template level.